### PR TITLE
[Scala] Scoping for the varargs type operator

### PR DESCRIPTION
**Pull request by djspiewak**
A suffix `*` is the syntax by which you declare varargs function parameters in Scala.  Confusingly, infix `*` is not a primitive operator, and may be declared in user-land.  So that makes the lookahead a bit tricky.

### CODE REVIEW

File: Scala/Scala.sublime-syntax [hunk 1]
```diff
@@ -390,6 +390,8 @@ contexts:
           push:
             - match: '{{typeprefix}}'
               set:
+                - match: '\*\s*(?=[\),=])'
+                  scope: keyword.operator.varargs.scala
```
Comment (wbond) [inline Scala/Scala.sublime-syntax]: Only the `*` should be scoped here, not the spaces.

File: Scala/Scala.sublime-syntax [hunk 2]
```diff
@@ -452,6 +454,8 @@ contexts:
...
```
Comment (wbond) [inline Scala/Scala.sublime-syntax]: Same here, use a capture so the spaces aren't scoped.

File: Scala/Scala.sublime-syntax (hunk 1 above)
Comment (djspiewak) [inline Scala/Scala.sublime-syntax]: Derp.  You're right of course.

**Comment by wbond** [comment]
Thanks!

**Pull request merged**